aws/signer/v2/v2_test.go

```diff
@@ -31,7 +32,7 @@ func (sb signerBuilder) BuildSigner() signer {
 		reader := bytes.NewReader(body)
 		req, _ = http.NewRequest(sb.Method, endpoint, reader)
 		req.Header.Add("Content-Type", "application/x-www-form-urlencoded")
-		req.Header.Add("Content-Length", string(len(body)))
+		req.Header.Add("Content-Length", strconv.Itoa(len(body)))
```

Comment:
one of best changes to vet in recent releases :)

Reply:
I swear I had fixed this in master, but it might have been after this branch was forked :D